build-system/app-index/index.js

```diff
@@ -34,31 +33,13 @@ const pc = process;
 // Sitting on /build-system/app-index, so we go back twice for the repo root.
 const root = path.join(__dirname, '../../');
 
-// JS Component
-const mainComponent = join(__dirname, '/components/main.js');
-
 // CSS
 const mainCssFile = join(__dirname, '/main.css');
 
-let shouldCache = false;
-function setCacheStatus(cacheStatus) {
-  shouldCache = cacheStatus;
-}
-
-
-let mainBundleCache;
-async function bundleMain() {
-  if (shouldCache && mainBundleCache) {
-    return mainBundleCache;
-  }
-  const bundle = await bundler.bundleComponent(mainComponent);
-  if (shouldCache) {
-    mainBundleCache = bundle;
-  }
-  return bundle;
+function setCacheStatus(unusedCacheStatus) {
+  // NOOP.
```

Comment:
This means you can remove `enable_dev_dashboard_cache` and all its associated logic, correct?